Source/Core/DiscIO/VolumeWad.cpp

```diff
@@ -60,16 +54,26 @@ IVolume::ECountry CVolumeWAD::GetCountry() const
 	if (!m_pReader)
 		return COUNTRY_UNKNOWN;
 
-	return CountrySwitch(m_Country);
+	// read the last digit of the titleID in the ticket
+	u8 country_code;
+	Read(m_tmd_offset + 0x0193, 1, &country_code);
```

Comment:
Possibly. Nearly all the TMD data locations come from WiiBrew, so it might not be to hard to do.